Dockerfile

```diff
@@ -6,13 +6,16 @@ ENV PYTHONUNBUFFERED=1
 # Install Django
 RUN pip install \
     coverage \
-    django==2.2 \
+    django \
     django-forms-bootstrap \
     django-leaflet \
     psycopg2 \
     thredds_crawler \
 &&  echo "alias ll='ls -lh'" >> /root/.bashrc
 
+# moving to django 3
+RUN pip install --upgrade django-cors-headers
```

Comment:
Why does it have to be a separate line? Can it be in the previous RUN ?

Reply:
Since it is an 'upgrade' not an 'install', it is better to separate it. But, I have tried to merge them into one line.

Reply:
If you specify the version (with the `==` syntax), you won't need this option, and then you can move this into the other `RUN` statement.
It's a good practice to limit the number of RUN, COPY and ADD statements in a Dockerfile, because each one creates an additional layer, which increases the size and decreases the performance of the image.
You can have a look [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers) for Dockerfile writing best practices.